internal/config/config.go

```diff
@@ -114,6 +114,15 @@ type HTTPBasicAuth struct {
 	Password string `mapstructure:"Password" tip:"Basic auth password"`
 }
 
+// Admin - Admin backend service configuration.
+type Admin struct {
+	ServerPort    int           `mapstructure:"ServerPort"`
+	HTTPAdminAuth HTTPAdminAuth `mapstructure:"HTTPAdminAuth"`
+	IssuerName    string        `mapstructure:"IssuerName"`
+	IssuerLogo    string        `mapstructure:"IssuerLogo"`
+	IssuerDID     string        `mapstructure:"IssuerDID"`
```

Comment:
We should add the "tip" tag. This is used by the experimental configurator to give a tip to the user about the purpose of this field

Reply:
done